osx/Scripts/build.sh

```diff
@@ -87,8 +87,8 @@ if [ -f "$helper" ]; then
   spctl --assess --verbose=4 $app_name.app
   echo "Checking Helper..."
   codesign -dvvvv $app_name.app/Contents/Library/LaunchServices/keybase.Helper
-  echo " "
-  spctl --assess --verbose=4 $app_name.app/Contents/Library/LaunchServices/keybase.Helper
+  # You don't spctl assess binaries anymore (only bundles)
```

Comment:
spctl is only supposed to apply to app bundles:
http://www.openradar.me/25618668

> Gatekeeper (as of 10.11.4) rejects anything that isn’t an app (or “like” an app, such a widget). This is part of a general hardening effort.

The codesign verify above does check the signature is correct.